docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -278,8 +292,8 @@ to Trino ``TIMESTAMP``.
 Mapping character types
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-Trino's ``VARCHAR(n)`` maps to ``VARCHAR2(n CHAR)`` if ``n`` is no greater than
-4000. A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
+Trino's ``VARCHAR(n)`` maps to ``VARCHAR2(n CHAR)`` if ``n`` is no greater
+than 4000. A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
```

Comment:
Thank you for saving so many potential wasted hours of future people who work on this document. ❤️